packages/xrpc-server/src/util.ts

```diff
@@ -60,7 +60,7 @@ export function decodeQueryParam(
   if (type === 'float') {
     return Number(String(value))
   } else if (type === 'integer') {
-    return Number(String(value)) | 0
+    return Number(String(value)) || 0
```

Comment:
I actually think this was a cast to integer!

Reply:
ah you're totally right

okay updated this to a `parseInt` instead which i think is best of both worlds